server/test/auth/StoredFileAclsTest.java

```diff
@@ -0,0 +1,33 @@
+package auth;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import models.Account;
+import org.junit.Test;
+import repository.ResetPostgres;
+import services.program.ProgramDefinition;
+import support.ProgramBuilder;
+
+public class StoredFileAclsTest extends ResetPostgres {
+
+  private ProgramDefinition programOne =
+      ProgramBuilder.newDraftProgram("program-one").buildDefinition();
+  private ProgramDefinition programTwo =
+      ProgramBuilder.newDraftProgram("program-two").buildDefinition();
+
+  @Test
+  public void hasProgramReadPermission_hasPermission() {
+    var acls = new StoredFileAcls().addProgramToReaders(programOne);
```

Comment:
Add a test for the empty acl case since all existing files will be in that state.

Reply:
done